index.js

```diff
@@ -287,40 +287,41 @@ client.on(Events.InteractionCreate, async (interaction) => {
                 // Fetch message tracker 
                 const messageTracker = await generalChat.messages.fetch(MESSAGE_TRACKER_ID);
 
-                // Grab current message tracker reviews 
-                const messageTrackerContent = messageTracker.content;
-
                 // Check if reviewer already has reviews in message tracker
-                const userIsAlreadyReviewer = checkForMatchInMessage(messageTrackerContent, reviewerId);
+                const userIsAlreadyReviewer = checkForMatchInMessage(messageTracker.content, reviewerId);
 
                 // Split current message tracker reviews line by line
-                const messageTrackerContentByLineArray = messageTrackerContent.split(LINE_REGEX);
+                const messageTrackerByLineArray = messageTracker.content.split(LINE_REGEX);
 
-                // Define non-default value for adding new reviewers and reviews
-                const isFirstBlock = false;
+                // Define review info object for passing to functions
+                const reviewInfo = {
```

Comment:
Thanks for making this into an object, that helps a lot with keeping things organized!